app/views/articles/_sidebar_campaign.html.erb

```diff
@@ -1,16 +1,19 @@
 <div class="widget">
+  <% if SiteConfig.campaign_sidebar_image %>
+    <img src="<%= SiteConfig.campaign_sidebar_image %>" />
+  <% end %>
   <header>
     <h4>
-      <% SiteConfig.campaign_featured_tags.each do |t| %>
-        <%= link_to("##{t}", "/t/#{t}") %>
-      <% end %>
+      <%= link_to "Stories (#{@campaign_articles_count})", "/t/#{SiteConfig.campaign_featured_tags.first}" %>
```

Comment:
I placed links to the first campaign tag here, this decision is on a discussion.